types/circular-json/index.d.ts

```diff
@@ -1,12 +1,11 @@
-// Type definitions for circular-json v0.1.6
+// Type definitions for circular-json v0.1.31
 // Project: https://github.com/WebReflection/circular-json
 // Definitions by: Jonathan Pevarnek <https://github.com/jpevarnek>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-
 interface ICircularJSON extends JSON {
```

Comment:
Is it necessary to have `extends JSON`? `interface JSON` just declares `parse` and `stringify` methods which are defined here anyway. It would be simpler to just have `export function parse(...)` and `export function stringify(...)` and not have to deal with `interface ICircularJSON` or `var CircularJSON`.

Reply:
I just fixed what was wrong with the old definitions and added a `tslint.json` for good measure, but I agree with you that it would be better to rewrite it from scratch. Should I?

Reply:
It's a 2-function module, might as well.

Reply:
Agreed